packages/tooling/openmrs/src/commands/assemble.ts

```
@@ -160,6 +161,7 @@ async function extractFiles(sourceFile: string, targetDir: string) {
   const packageJson = JSON.parse(
     files[`${packageRoot}/package.json`].toString("utf8")
   );
+  const version = packageJson.version ?? "unknown";
```

I'd rather prefer something like `0.0.0` as its still a valid version number, but indicating that something is missing.

With the given value we have an invalid version - i.e., some tools might panic here. If this is what we want I'd say let's throw an error right away instead of failing with these tools - or have a proper version as a fallback value here.